### ISSUE TITLE
Is there a global setting for row height in the data browser?

### ISSUE THREAD

**Issue by JoanEliot** [sqlitebrowser/sqlitebrowser#2081]
What is your question
-----------------------
Sorry if I'm missing where these issues are addressed in the docs.

**Comment by MKleusberg** [comment]
No, there's no such option at the moment. The row height does depend on the selected font size however and is chosen to fit a single row by default. Can you provide some details about what you're trying to achieve? I'm asking because we're trying to limit the number of settings to a manageable amount for our small dev team :smile:

**Comment by mgrojo** [comment]
In the nightly there is an option for automatically adjusting the row heights to the contents. I don't know if that is what you want, but you can take a look at #2006 for the description of the feature.
